doc/tutorial/introduction/index.md

```diff
@@ -130,7 +132,7 @@ argument to your program (e.g. call `./myprogram seq.fasta`).
 \snippet introduction_read_fasta.cpp read
 \endsolution
 
-Note that the same code can also read FastQ files and the `qual` variable will not be empty then. If you like, try it!
+Missing `include <vector>` and `include <string>`? As shown above, they are already part of seqan3! Note that the same code can also read FastQ files and the `qual` variable will not be empty then. If you like, try it!
```

Comment:
We have the policy that _in tutorials and snippets_ we state all includes that are needed directly from the functions in the code snippet. Although they can be technically omitted, they demonstrate from which header files the functions or data structures are used. Therefore, please remove this sentence and adapt the includes accordingly.

Reply:
According to this, I added #include <filesystem> in two files. One passed the tests (introduction_file_input), but the other not (introduction_align). I knew it's new in c++ 17, but why did one pass and one not? And am I to skip it or leave it in?

Reply:
Do you refer to the filesystem include? Please use instead:
```
#include <seqan3/std/filesystem>
```